include/tof_ros2cpp/aditof_utils.h

```diff
@@ -53,13 +53,13 @@ std::string * parseArgs(int argc, char ** argv);
 std::shared_ptr<aditof::Camera> initCamera(std::string * arguments);
 void startCamera(const std::shared_ptr<aditof::Camera> & camera);
 void stopCamera(const std::shared_ptr<aditof::Camera> & camera);
-void setFrameType(const std::shared_ptr<aditof::Camera> & camera, const std::string & type);
+void setCameraMode(const std::shared_ptr<aditof::Camera> & camera, const uint8_t & mode);
```

Comment:
Do we need both setMode and setCameraMode?

Reply:
I removed setMode because is the same as setCameraMode